corehq/apps/app_manager/models.py

```diff
@@ -5143,7 +5143,7 @@ def make_build(self, comment=None, user_id=None, previous_version=None):
         if not copy._id:
             # I expect this always to be the case
             # but check explicitly so as not to change the _id if it exists
-            copy._id = copy.get_db().server.next_uuid()
+            copy._id = uuid.uuid4().hex
```

Comment:
The format is the same:
```
aef4b1631098306c1d7239808c000871 # couch
29a07f566f854b089f5a1bda057dba9d # python
```
but the algorithm is different. Couch uses a sequential algorithm to generate unique ID's per couch instance: http://docs.couchdb.org/en/2.0.0/config/misc.html#uuids